sample/src/main/java/org/aerogear/graphqlandroid/activities/MainActivity.kt

```diff
@@ -77,13 +102,91 @@ class MainActivity : AppCompatActivity() {
 
         pull_to_refresh.setOnRefreshListener {
             doYourUpdate()
-
+            pull_to_refresh.isRefreshing = false
         }
 
         recycler_view.layoutManager = LinearLayoutManager(this)
         recycler_view.adapter = taskAdapter
 
-        fabAdd.setOnClickListener {
+        //Used for in-memory mutations
+        buttonOffline.setOnClickListener {
+
+            Log.e(TAG, "buttonOffline clicked")
+            OfflineMutationSender()
+        }
+
+        //Used for persistent mutations
+        persistence.setOnClickListener {
+
+
+            Log.e(TAG, "in persistence")
+            val listOfMutations = dbDao.getAllMutations()
+            Log.e(TAG, "in persistence size of list : ${listOfMutations.size}")
+
+            listOfMutations.forEach { storedmutation ->
+
+                val responseClassName = storedmutation.responseClassName
+
+                val classReflected: Class<*> = Class.forName(responseClassName)
+
+                val constructor = classReflected.constructors.first()
```

Comment:
Maybe add some comments for the implementation here? 
Outlinign why we only took the first constructor?